src/BenchmarksApps/TodosApi/Program.cs

```diff
@@ -0,0 +1,73 @@
+using Microsoft.Extensions.Logging.Configuration;
+using Npgsql;
+using TodosApi;
+
+var builder = WebApplication.CreateSlimBuilder(args);
+
+#if ENABLE_LOGGING
```

Comment:
Seems weird that `ENABLE_LOGGING` turns off appsettings.json, which is needed for AddJwtBearer right?

Reply:
Just fixed that :)